packages/tsdocs/package.json

```diff
@@ -13,7 +13,8 @@
     "extract-apidocs": "node bin/extract-apis",
     "document-apidocs": "api-documenter markdown -i ../../docs/apidocs/models -o ../../docs/site/apidocs",
     "update-apidocs": "node bin/update-apidocs",
-    "build": "lb-tsc && npm run build --prefix fixtures/monorepo",
+    "build": "lb-tsc",
+    "build:fixtures": "npm run build --prefix fixtures/monorepo",
```

Comment:
Do we want to run it as part the `npm run build:full` for `loopback-next`? I'm worried that we forget to rebuild the fixtures during TypeScript upgrade.

Reply:
Sure, why not. 

I feel such change is out of scope of this pull request though.

In this PR, I am removing the inconsistency where `npm run build` in monorepo does not build tsdocs fixtures while `npm run build` in `packages/tsdocs` does.

I'd like to land this PR first, and then discuss what process we want to use to ensure fixtures are rebuilt after TypeScript upgrades.

One option that comes to my mind is to re-build the fixtures from `postinstall` hook, which is honored by `lerna bootstrap` and is not executed as often as the build task.